### DISCUSSION TITLE
在一个ArrayField下array map生成一组Field,在每一个Field上设置required，即使输入了正确的数字也会报错，如何处理？

### DISCUSSION THREAD

**Discussion by specialCoder** [alibaba/formily#1668] [Q&A 2.x]
<ArrayField
        name="numbers"
      >
      {
        (field) => {
           return (field.value || []).map(({ week = '-', day = '-'},i) => {
              return <Field
                key={i}
                required
                name={`W${week}D${day}`}
                title={`W${week}D${day}`}
                decorator={[FormItem]}
                component={[NumberPicker,{
                  precision:0,
                }]}
              />
            })
        }
      }
      </ArrayField>

**Comment by xiaoxiansheng1** [comment]
请问找到解决方法了吗？也遇到同样问题。
